src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,34 @@
+ApiClient_.prototype.fetchGetMetric = function (metric, slug, from, to, timeBoundSuffix) {
+  var timeBound = ['', '1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
```

Comment:
I think `''` should not be in this list - you're already guarding against that with `if (timeBoundSuffix !== '') {`, so you'll never attempt to fetch it